packages/button/src/styles/Button.css

```diff
@@ -400,3 +400,144 @@
   width: 2.4rem;
 }
 /* stylelint-enable no-descending-specificity */
+
+/* ------- PRIMARY FLAIR ------- */
+
+.Button--primaryFlair {
+  background: var(
+    --lp-color-bg-interactive-primary-flair-base-dark,
+    linear-gradient(124deg, #a34fde 0%, #405bff 98.05%)
+  );
+  border-radius: var(--lp-radius-medium, 6px);
+  border: 1px solid var(--Button-color-border-primary-flair);
+  color: var(--Button-color-text-primary);
+}
+
+.Button-icon.Button--primaryFlair {
+  fill: var(--Button-color-text-primary);
+}
+
+.Button-icon.Button--icon.Button--primaryFlair {
+  background-color: transparent;
+  color: var(--Button-icon-color-text-default);
+}
+
+.Button--primaryFlair:hover,
+.Button--icon.Button--primaryFlair:hover {
+  background: var(
+    --lp-color-bg-interactive-primary-flair-hover-dark,
```

Comment:
This will apply to all gradient values
https://github.com/launchdarkly/launchpad-ui/pull/977/files#r1303348140